NEWS.md

```diff
@@ -1,135 +1,15 @@
 ======
 
 ## Highlights
 
- * MQTT destination
-
-   The new `mqtt()` destination can be used to publish messages using the MQTT protocol.
-   Currently MQTT 3.1.1 and 3.1 are supported.
-
-   Supported transports: `tcp`, `ws`.
-
-   Example config:
-   ```
-   destination {
-     mqtt{
-       address("tcp://localhost:1883"),
-       topic("syslog/$HOST"),
-       fallback-topic("syslog/fallback"))
-     };
-   };
-   ```
-
-   Note: MQTT 5.0 and TLS (`ssl://`, `wss://`) are currently not supported.
-   ([#3703](https://github.com/syslog-ng/syslog-ng/pull/3703))
-
- * `discord()` destination
-
-   syslog-ng now has a webhook-based Discord destination.
-   Example usage:
-   ```
-   destination {
-     discord(url("https://discord.com/api/webhooks/x/y"));
-   };
-   ```
-
-   The following options can be used to customize the destination further:
-
-   `avatar-url()`, `username("$HOST-bot")`, `tts(true)`, `template("${MSG:-[empty message]}")`.
-   ([#3717](https://github.com/syslog-ng/syslog-ng/pull/3717))
-
-## Features
-
- * kafka-c: batching support in case of sync-send(yes)
-
-   ```
-   kafka-c(
-    bootstrap-server("localhost:9092")
-    topic("syslog-ng")
-    sync-send(yes)
-    batch-lines(10)
-    batch-timeout(10000)
-   );
-   ```
-
-   Note1: batch-lines are accepted in case of sync-send(no), but no batching is done.
-   Note2: messages are still sent one at a time to kafka, the batch yields multiple message per transaction.
-   ([#3699](https://github.com/syslog-ng/syslog-ng/pull/3699))
-
- * kafka-c: sync-send(yes) enables synchronous message delivery, reducing the possibility of message loss.
-
-   ```
-   kafka-c(
-     bootstrap-server("localhost:9092")
-     topic("syslog-ng")
-     sync-send(yes)
-   );
-   ```
-
-   Warning: this option also reduces significantly the performance of kafka-c driver.
-   ([#3681](https://github.com/syslog-ng/syslog-ng/pull/3681))
-
- * `disk-buffer`: Now we optimize the file truncating frequency of disk-buffer.
-
-   The new behavior saves IO time, but loses some disk space, which is configurable with a new option.
-   The new option in the config is settable at 2 places:
-   * `truncate-size-ratio()` in the `disk-buffer()` block, which affects the given disk-buffer.
-   * `disk-buffer(truncate-size-ratio())` in the global `options` block, which affects every disk-buffer
-     which did not set `truncate-size-ratio()` itself.
-   The default value is 0.01, which operates well with most disk-buffers.
-
-   If the possible size reduction of the truncation does not reach `truncate-size-ratio()` x `disk-buf-size()`,
-   we do not truncate the disk-buffer.
-
-   To completely turn off truncating (maximal disk space loss, maximal IO time saved) set `truncate-size-ratio(1)`,
-   or to mimic the old behavior (minimal disk space loss, minimal IO time saved) set `truncate-size-ratio(0)`.
-   ([#3689](https://github.com/syslog-ng/syslog-ng/pull/3689))
-
+<Fill this block manually from the blocks below>
```

Comment:
Should we put the disk-buffer bugfix in highlights too?

Reply:
IMHO, it's fine without Highlights, see: https://github.com/syslog-ng/syslog-ng/releases/tag/syslog-ng-3.31.2

Reply:
I think we should just remove the highlight section. We fixed a disk-buffer regression, that's not a bright news in my opinion.